test/coverband/configuration_test.rb

```diff
@@ -86,4 +86,23 @@ def setup
       end
     end
   end
+
+  test 'use_oneshot_lines_coverage' do
+    Coverband::Collectors::Coverage.instance.reset_instance
```

Comment:
Do we need to reset here? Was under impress that our setup does this (I hope).

Reply:
Removed